<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>window</class>
 <widget class="QMainWindow" name="window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>495</width>
    <height>609</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>window</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="vendorTabs">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>471</width>
      <height>521</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(/img/coin.png);</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="customerInterface">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QWebView" name="vendorInner">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>110</y>
        <width>361</width>
        <height>351</height>
       </rect>
      </property>
      <property name="url">
       <url>
        <string>about:blank</string>
       </url>
      </property>
     </widget>
     <widget class="QPushButton" name="refundButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>410</y>
        <width>61</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:red</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="productFourButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>290</y>
        <width>61</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>4</string>
      </property>
     </widget>
     <widget class="QPushButton" name="productTwoButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>61</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
     <widget class="QPushButton" name="productThreeButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>230</y>
        <width>61</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>3</string>
      </property>
     </widget>
     <widget class="QPushButton" name="productFiveButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>350</y>
        <width>61</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>5</string>
      </property>
     </widget>
     <widget class="QPushButton" name="insertCoinButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>71</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: gold;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="vendorDisplayStatus">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>30</y>
        <width>361</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: black; color: lime;  padding: 5px; font: 11pt &quot;Digital Numbers&quot;;</string>
      </property>
      <property name="text">
       <string>Status</string>
      </property>
     </widget>
     <widget class="QLabel" name="vendorDisplayDetails">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>60</y>
        <width>361</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: black;  color:lime;  padding: 5px; qproperty-alignment: AlignRight;
font: 11pt &quot;Digital Numbers&quot;;</string>
      </property>
      <property name="text">
       <string>Detailinformation</string>
      </property>
     </widget>
     <widget class="QPushButton" name="productOneButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>61</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="machineInner">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>495</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>?</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKitWidgets/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
